setup.py

```diff
@@ -170,10 +170,12 @@ def run(self):
             in_files.append(
                 os.path.join(SRC_DIR, '{}.in.proto'.format(stem)))
             out_files.extend([
-                os.path.join(SRC_DIR, '{}.proto'.format(stem)),
-                os.path.join(SRC_DIR, '{}.proto3'.format(stem)),
-                os.path.join(SRC_DIR, '{}-ml.proto'.format(stem)),
-                os.path.join(SRC_DIR, '{}-ml.proto3'.format(stem)),
+                os.path.join(SRC_DIR, '{}.pb.h'.format(stem)),
+                os.path.join(SRC_DIR, '{}-ml.pb.h'.format(stem)),
+                os.path.join(SRC_DIR, '{}_{}.proto'.format(stem, ONNX_NAMESPACE)),
```

Comment:
In case of ONNX_NAMESPACE is the default value (onnx), I would like to preserve the output .proto files name as `onnx.proto`, otherwise development git checkout of the onnx repo will almost always have some random .proto files showing as untracked. It's also easier to see whether we need to update the generated (and checked in) onnx.proto files.

Reply:
Hmm, why do we have `onnx.proto` in our src file in the first place? It is supposed to be generated.

Reply:
it's easier for users to just grab the onnx.proto file and use it for their own purpose. Not all use cases will need to have their own namespace.